build_script/publications.gradle

```diff
@@ -61,13 +61,19 @@ def manipulatePomXml = {
 publishing {
   publications {
     // Define the "release" publication
-    release(MavenPublication) {
-      from components.java
+    release(MavenPublication) { publication ->
+      if (project.name == 'restli-netty') {
+        apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
Why are we applying a plugin in the Maven publication closure? Seems pretty weird... Also, isn't this plugin already applied in this project's `build.gradle` file?

Reply:
This does seem to be required here for running the `local-release` script or the attempt to access the `shadow` property fails. If only for a normal release/publish case, I did not see an issue with omitting the plugin apply here.